cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/aws/wizard-basic/management-credential-step/ManagementCredentials.tsx

```diff
@@ -68,10 +70,19 @@ function ManagementCredentials(props: Partial<StepProps>) {
     }, []);
 
     useEffect(() => {
+        const fetchKeyPairs = async () => {
```

Comment:
Is there any way to reuse this below (for the refresh)?

Reply:
good suggestion. move the api out of the scope.